src/RA_Core.cpp

```diff
@@ -500,10 +502,11 @@ API BOOL CCONV _RA_OfferNewRAUpdate(const char* sNewVer)
 {
     char buffer[1024];
     sprintf_s(buffer, 1024, "Update available!\n"
-        "A new version of %s is available for download at " RA_HOST_URL ".\n\n"
+        "A new version of %s is available for download at %s.\n\n"
         "Would you like to update?\n\n"
         "Current version:%s\n"
         "New version:%s\n",
+        Narrow(RA_HOST_URL).data(),
```

Comment:
Prefer `c_str()` for consistency.

Reply:
I'm sorry, when I saw the warning in VS it said it expected char*. As long as std::string isn't const data() returns char* and c_str() returns const char* (C++17) but it doesn't matter in this case so it can be const char*.